src/components/TokenInput/index.tsx

```
@@ -395,7 +402,60 @@ const Swap: FC<SwapProps> = ({
       setAmount(balance);
     }
   };
-  const pools: PoolI[] = useSelector((state: RootState) => state.pools.pools);
+  const isWVOI = tokInfo?.tokenId === "0";
+  const badge = isWVOI ? (
+    <Tooltip title="Trusted by Nautilus" placement="right" arrow>
+      <VerifiedUserIcon fontSize="small" sx={{ color: "gold " }} />
+    </Tooltip>
+  ) : tokInfo?.verified || 0 > 0 ? (
```

Biome

[error] Comparing to itself is potentially pointless.



(lint/suspicious/noSelfCompare)